package.json

```diff
@@ -19,7 +19,7 @@
     "install:all": "yarn --cwd app && yarn",
     "dev": "cross-env DEBUG=true NACL_FAST=disable NODE_OPTIONS='--openssl-legacy-provider' webpack serve --config ./setup/config/webpack.config.dev",
     "dev:hw": "cross-env DEBUG=true webpack serve --config ./setup/config/webpack.config.dev --https",
-    "start": "cross-env DEBUG=true yarn rebuild && electron ./app/",
+    "start": "cross-env DEBUG=true && yarn rebuild && electron ./app/",
```

Comment:
I add the && operator here because its `DEBUG=true` doesn't seem to be set when the start script is ran